osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -17,8 +19,8 @@ public class UpdateSettings : SettingsSubsection
 
         protected override string Header => "Updates";
 
-        [BackgroundDependencyLoader]
-        private void load(Storage storage, OsuConfigManager config, OsuGameBase game)
+        [BackgroundDependencyLoader(true)]
+        private void load(Storage storage, OsuConfigManager config, OsuGame game)
```

Comment:
What is this for?

Reply:
I'm pretty sure that this is another PR that got merged (it doesn't get highlighted as my change on Files Changed).

Reply:
Weird, how did this got displayed to me as if it was your change o_O